mypy/build.py

```diff
@@ -837,6 +837,8 @@ def parse_file(
             self.errors.ignored_files.add(path)
         tree = parse(source, path, id, self.errors, options=options)
         tree._fullname = id
+        if options.use_builtins_fixtures and id in CORE_BUILTIN_MODULES:
+            tree._is_typeshed_file = True
```

Comment:
I'd rather not do this if this can silence errors if test fixtures have issues. It would be better to either add a minimal custom fixture that is only used in the relevant test cases or to omit the test cases that need this (I trust that you've tested it already).

Reply:
`_is_typeshed_file` does not silence all errors, though.

Reply:
The premise of errors in fixtures becoming obscured concerned me too, but having the test environment closer to actual environment seemed more prudent.

If you feel particularly iffy about it, sure, I can add `# type:ignore[misc]` to a handful of places in the fixtures.